golang/src/image-janitor/main.go

```diff
@@ -0,0 +1,255 @@
+package main
+
+import (
+	"bytes"
+	"configurate"
+	"dockerops"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"logcabin"
+	"model"
+	"os"
+	"path"
+	"regexp"
+	"strings"
+	"time"
+)
+
+var (
+	version       = flag.Bool("version", false, "Print version information.")
+	interval      = flag.String("interval", "1m", "Time between clean up attempts.")
+	cfgPath       = flag.String("config", "/etc/jobservices.yml", "Path to the config.")
+	readFrom      = flag.String("read-from", "/opt/image-janitor", "The directory that job files are read from.")
+	dockerURI     = flag.String("docker", "unix:///var/run/docker.sock", "The URI for connecting to docker.")
+	gitref        string
+	appver        string
+	builtby       string
+	filenameRegex = regexp.MustCompile(`(?i)^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}\.json$`)
+)
+
+func init() {
```

Comment:
There's no `logcabin.Init` done here like in the other go services, so I'd suspect the logging doesn't work correctly.